### PR TITLE
Capstone Java script Project

### PR DESCRIPTION
**Pull request by AsimKhan2019**
For this Capstone Project we implemented the following tasks: 

Created a front page with the items displayed from the API, and the number of items counter next to the Launches link, as this was a NASA space launch detail API. 
We implemented the likes and the comments' actions for the project and displayed the counter for both likes and comments. 
We also implemented adding the comments function to each launch detail. 
Using the Involvement API, we were able to record our likes and comments to the API. 
We also implemented the tests using jest for the counters, although the instructions were not clear. The test itself was a difficult implementation. 
The reservation was not part of our task, as we were only 2 students, so I have closed those issues so that we could be more focused on our tasks.
We also implemented the design as best to the time frame provided it was not an easy project as the purpose was to implement JavaScript milestones and API implementations. This project itself was very long to implement, we did face problems along the way, and we had plenty of blockers. 
The responsive requirement was additional/optional. Though we were short on time and tried to implement the APIs and functions, we managed to make the presentation better by the wire frame.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Kanban Board
 
 > This project is all about using Kanban Board and applying the right GitFlow approach to develop the project. The 
-project is all about using APIs and retrieving and postinjg comments and data using several different APIs and integrations. 
+project is all about using APIs and retrieving and postinjg comments and data using several different APIs and integrations. Creating new branches from different commits using different heads
```
Comment (AsimKhan2019) [inline README.md:6]: Updated content for the Read Me file.

File: src/modules/launchList.js [hunk 2]
```diff
@@ -128,6 +128,9 @@ export default class Launch {
   }
 
   static async getComments() {
+    const getDetails = document.querySelector('.commentList');
+        getDetails.innerHTML = ' ';
+        commentsCount.innerText = '0';
```
Comment (AsimKhan2019) [inline src/modules/launchList.js]: The implementation here is to clear the comments and set counter to 0 if there are no comments.

File: src/modules/launchList.js [hunk 3]
```diff
@@ -139,15 +139,12 @@ export default class Launch {
         return response.json();
       })
       .then((json) => {
-        
-       
-          commentsCount.innerText = json.length; // if json.length > 0 then do all this else no 
+        commentsCount.innerText = json.length; // if json.length > 0 then do all this else no
```
Comment (AsimKhan2019) [inline src/modules/launchList.js]: This is for the comments counter to get the total count for the comments

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @AsimKhan2019 and @ifzyy, 

## Status: APPROVED 🥳 🥳 

image

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

- [ ] Please make sure you add your presentation video to your Readme or pull request. It can easily be missed if it's just in the additional information section 🙏 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AsimKhan2019** [comment]
@Ugboaja-Uchechi Thank you for the approval.

**Pull request merged**